.github/workflows/ci.yml

```diff
@@ -57,6 +57,20 @@ jobs:
           else
               echo 'Not going to push build'
           fi
+  windows-mingw:
+    name: Windows (win32) using mingw
+    runs-on: ubuntu-latest
+    container:
+      image: openrct2/openrct2-build:0.2.4-mingw
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
```

Comment:
v2 should be used.

Reply:
I had to look it up, so fwiw: https://github.com/actions/checkout#checkout-v2